bxml/buildingBlocks/createCall.md

```diff
@@ -0,0 +1,20 @@
+# Create an Outbound Call
+
+## Concept
+
+To create an outbound call, we will use the RestAPI. The biggest difference is the callbackHttpMethod must be <code class="get">GET</code>. For more information about creating an outbound call, visit the [Full API Reference](../../methods/calls/postCalls.html)
+
+## Code
+
+The from number is a Bandwith number you own. Change the `to`, `from`, and `callbackUrl` parameters.
+
+```js
+const callParameters = {
+    to                 : +12345678901,
+    from               : +12345678901,
+    callbackUrl        : callbackUrl,
+    callbackHttpMethod : 'GET'
+}
+const call = await client.Call.create(callParameters);
+res.status(201).send(call);
+```
```

Comment:
In the credentials overview you had used `var` instead of `const` or `let`. All good, just choose one pattern and be consistent (I've had a bad time with this recently myself)

Read about the [differences](https://dev.to/sarah_chima/var-let-and-const--whats-the-difference-69e).

### Code typo

```js
const callParameters = {
    to                 : +12345678901,
    from               : +12345678901,
    callbackUrl        : callbackUrl,
    callbackHttpMethod : 'GET'
}
``` 

> Should be

```
const callParameters = {
    to                 :  "+12345678901",
    from               : "+12345678902",
    callbackUrl        : "http://your-callback-url.com/outbound-call-events",
    callbackHttpMethod : "GET"
}
```